src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -41,8 +41,8 @@
     public static final String VALID_TASK_MODEL = "model";
     //TODO: after Parsing of tasks is refined, may update these to be more meaningful
 
-    public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
-    public static final String NAME_DESC_BOB = " " + PREFIX_NAME + VALID_NAME_BOB;
+    public static final String NAME_DESC_AMY = " " + PREFIX_PROJECT_NAME + VALID_NAME_AMY;
+    public static final String NAME_DESC_BOB = " " + PREFIX_PROJECT_NAME + VALID_NAME_BOB;
```

Comment:
Is it better to use NAME_DESC_A instead of NAME_DESC_AMY?

Reply:
Yes indeed, I will also change the project names to more appropriate "project-like" names too. But for now I just want to get the Name refactored and to fix all the errors. On the third PR when I refactor the Address and tags, I will change all the testcases and such. 

If I change it now, it will be a lot more time-consuming.